lemmy.yml

```diff
@@ -62,13 +62,12 @@
   - name: Install Docker Module for Python
     pip:
       name: docker
+      state: latest
 
   - name: Install or upgrade docker-compose
-    get_url: 
-      url : "https://github.com/docker/compose/releases/download/1.25.5/docker-compose-Linux-x86_64"
-      dest: /usr/local/bin/docker-compose
-      mode: 'a+x'
-      force: yes
+    pip:
+      name: docker-compose
+      state: latest
```

Comment:
You should be able to install multiple packages in a single command like this:
```
- name: Install multi python packages with version specifiers
  ansible.builtin.pip:
    name:
      - django>1.11.0,<1.12.0
      - bottle>0.10,<0.20,!=0.11
```

https://docs.ansible.com/ansible/latest/collections/ansible/builtin/pip_module.html

Reply:
I kept them in their own tasks purely to avoid altering the task structure.  I've pushed an updated commit to combine these in to a single task.  Thanks for the review.